client/src/modules/pavillion/modals/createUpdate.html

```diff
@@ -0,0 +1,58 @@
+<form name="PavillionForm" bh-submit="ModalCtrl.submit(PavillionForm)" novalidate>
+  <div class="modal-header">
+  <ol class="headercrumb">
+    <li ng-if="ModalCtrl.isCreating" class="title">
+      <span translate>FORM.LABELS.PAVILLION</span>
+      <label class="badge badge-warning" translate>FORM.LABELS.CREATE</label>
+    </li>
+    <li ng-if="!ModalCtrl.isCreating" class="title">
+      <span translate>FORM.LABELS.PAVILLION</span>
+      <label class="badge badge-warning" translate>FORM.LABELS.UPDATE</label>
+    </li>
+  </ol>
+</div>
+
+  <div class="modal-body">
+    <div class="form-group" ng-class="{ 'has-error' : PavillionForm.$submitted && PavillionForm.name.$invalid }">
+      <label class="control-label" translate>FORM.LABELS.NAME</label>
+      <input name="name" ng-model="ModalCtrl.pavillion.name" autocomplete="off" class="form-control" required>
+      <div class="help-block" ng-messages="PavillionForm.text.$error" ng-show="PavillionForm.$submitted">
+        <div ng-messages-include="modules/templates/messages.tmpl.html"></div>
+      </div>
+    </div>
+
+    <div class="form-group"
+      ng-class="{'has-error' : ModalForm.$submitted && ModalForm.service.$invalid }">
+      <label class="control-label" translate>FORM.LABELS.SERVICE</label>
+      <select
+        class="form-control"
+        name="service"
+        ng-model="ModalCtrl.pavillion.service_id"
+        ng-options="service.id as service.name for service in ModalCtrl.services"
```

Comment:
Can we use [bhServiceSelect](https://github.com/IMA-WorldHealth/bhima/blob/master/client/src/js/components/bhServiceSelect.js) here?